ultralytics/engine/exporter.py

```diff
@@ -359,12 +359,12 @@ def get_int8_calibration_dataloader(self, prefix=""):
             task=self.model.task,
             imgsz=self.imgsz[0],
             augment=False,
-            batch_size=self.args.batch * 2,  # NOTE TensorRT INT8 calibration should use 2x batch size
```

Comment:
@ambitious-octopus this comment here states that we need a 2x batch size. I think this change will crash INT8 TRT export.

Reply:
The problem here is that this function (`get_int8_calibration_dataloader`) is also used for `INT8` quantization with OpenVINO, causing a hard-coded increase in the batch size (2*). This explains why line 207 forces the model to be dynamic, otherwise OpenVINO would not work.